uv/manager/manager.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
```

Comment:
call this file `impl` if you want to keep it nested, for `implementation`. otherwise we have a triple use of manager...

```python
uv.manager.manager.MeasurementManager
```

Reply:
Done